Sources/MapboxDirectionsCLI/main.swift

```diff
@@ -4,14 +4,13 @@ import Foundation
 import MapboxDirections
 import ArgumentParser
 
-
 struct ProcessingOptions: ParsableArguments {
     
     @Option(name: [.short, .customLong("input")], help: "[Optional] Filepath to the input JSON. If no filepath provided - will fall back to Directions API request using locations in config file.")
     var inputPath: String?
     
-    @Option(name: [.short, .customLong("config")], help: "Filepath to the JSON, containing serialized Options data.")
-    var configPath: String
+    @Argument(help: "Path to a JSON file containing serialized RouteOptions or MatchOptions properties, or the full URL of a Mapbox Directions API or Mapbox Map Matching API request.")
```

Comment:
This is kind of unwieldy as an argument description. A possible future improvement would be to accept multiple configurations, perhaps a mix of files and URLs, and merge them somehow.

Reply:
will you document that possible future improvement?

Reply:
Let’s track it as tail work in #729.